src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -657,6 +665,16 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
             return;
         }
 
+        /**
+         * Track user leaves the image
+         */
+        if (this.state.selectedAsset && this.state.selectedAsset.asset) {
```

Comment:
You refer to `this.state.selectedAsset` quite often.
Maybe destructuring
```
const { selectedAsset } = this.state
```
would make it slightly more readable?

Same applies to props.